app/routes/create-catch-certificate/$catchCertificate/what-are-you-exporting.tsx

```diff
@@ -114,11 +136,10 @@ const AddSpeciesPage = () => {
               states={stateLookup.states}
               presentations={stateLookup.presentations}
               commodityCodes={commodityCodes}
-              faoCode={faoCode}
+              faoCode={_faoCode || faoCode}
```

Comment:
exactly my previous point... what is the difference between _faoCode and faoCode?